sites/all/themes/cetaviano/sass/partials/utils/_variables.scss

```diff
@@ -64,6 +64,6 @@ $site-breakpoints: (
 $space-x-small: 1em; //16px
```

Comment:
@Marymartinezb 
Hay un pequeño gran problema. Los comentarios estan mal, o el uso de las variables esta mal, porque, en varios lugares he visto que usan estos valores no solo para "between titles and content" sino para margin y/o padding. Ojo, el comentario **no es para parar el merge de esto**, de hecho, hagan el merge, pero,  deberian discutir si valen la pena los comentarios en estas variables, o si el uso propuesto lo esta siguiendo el equipo

Reply:
@jleiva creo que los comentarios son los que estarían mal, eran para guiarnos con las diferentes medidas y en el diseño en que se estaban usando.
Pero como son medidas estándar deberían poder ser utilizadas para el responsive y diferentes espaciados.

Reply:
Copiado, mergealo; pero seria bueno eliminar comentarios que induzcan a error

Reply:
Entendido, en cuanto sigamos trabajando en estilos los borramos